mcs/resource_mcs_db_instance.go

```diff
@@ -0,0 +1,695 @@
+package mcs
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+// DBInstanceDelay ...
+const (
+	DBInstanceDelay      = 10
```

Comment:
could declare as a time type from the beginning

Reply:
We followed kubernetes_cluster style, and it also declares these constants as integers:
https://github.com/MailRuCloudSolutions/terraform-provider-mcs/blob/f154f5d3e75b71dfde21a7ed5e8fa26ba2ab65a2/mcs/resource_mcs_kubernetes_cluster.go#L14

Reply:
Fixed